Frameworks/include/CGImageInternal.h

```diff
@@ -252,6 +184,17 @@ struct __CGImage : CoreFoundation::CppBase<__CGImage, __CGImageImpl> {
         return _impl.bitmapImageSource;
     }
 
+    inline void* Data() const {
+        Microsoft::WRL::ComPtr<IWICBitmapLock> lock;
+        IWICBitmap* bitmap = dynamic_cast<IWICBitmap*>(_impl.bitmapImageSource.Get());
```

Comment:
same as before, i'll update.

---
In reply to: [86920203](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r86920203) [](ancestors = 86920203)